Inspicio/Controllers/ImagesController.cs

```diff
@@ -93,12 +93,13 @@ public IActionResult Create()
         }
 
 
-        public class DataToBody
+        public class FullReview
         {
             public Image Image { get; set; }
-            public List<Comment> Comments { get; set; }
+            public List<Tuple<Comment, String>> Comments { get; set; }
```

Comment:
Tuples can get a bit confusing if they're not used straight away, you end seeing code like `comment.Item1` and are left wondering what it means, so I'd avoid using them for properties if I can. In a case like this, you could just go with a nested class to give the list elements a more informative name. e.g. 
```
public class FullReview
{
    public class UserComment
    {
        public string User { get; set; }
        public Comment Comment { get; set; }
    }

    public Image Image { get; set; }
    public List<UserComment> Comments { get; set; }
}
```

I might have misunderstood what the string means though (I haven't tried running the app yet)